### ISSUE TITLE
Show SHA of the commit of the build from source

### ISSUE THREAD

**Issue by BanzaiMan** [php-build/php-build#356]
When building from source (`master`, `PHP-7.0` and so on), it would be instructive to show what SHA is fetched and built.

Currently, I see the nightly builds failing on the `master` (https://travis-ci.org/travis-ci/php-src-builder/jobs/97558831#L237), but it is difficult to match it up against https://github.com/php/php-src and report it.

**Comment by rogeriopradoj** [comment]
Hi, @BanzaiMan, how are you?

Sorry for taking so long to getting in touch. I think your idea would be a great addition! Would you like to send a PR to the project with the implementation that fits your needs in Travis?

**Comment by GrahamCampbell** [comment]
Any movement here?

**Comment by hnw** [comment]
Hi. 

I implemented my idea against this issue in #458.
After cloning from GitHub, commit hash will be displayed as follows:

```
$ ./bin/php-build master /tmp/foobarbaz
[Info]: Loaded extension plugin
[Info]: Loaded apc Plugin.
[Info]: Loaded composer Plugin.
[Info]: Loaded github Plugin.
[Info]: Loaded uprofiler Plugin.
[Info]: Loaded xdebug Plugin.
[Info]: Loaded xhprof Plugin.
[Info]: Loaded zendopcache Plugin.
[Info]: php.ini-production gets used as php.ini
[Info]: Building master into /tmp/foobarbaz
[Cloning]: Branch master from git://github.com/php/php-src.git
[Cloning]: commit 7f69b2b499251f596e407776d5a6ed4545398a9a

(snip)
```

What do you think of this?
